From 0f5a8899370416c275e6bd95f6c397250d7d07e6 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Mon, 18 Aug 2014 19:55:51 -0400
Subject: [scsi] qla2xxx: Correction to message ids

Message-id: <1408391817-6389-4-git-send-email-cdupuis@redhat.com>
Patchwork-id: 87902
O-Subject: [PATCH RHEL 7.1 e-stor 03/69] qla2xxx: Correction to message ids.
Bugzilla: 1089346
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Saurav Kashyap <saurav.kashyap@qlogic.com>

scsi commit id 6ddcfef7e10b5e6faa05533674fcec4a7298ad3a

Signed-off-by: Giridhar Malavali <giridhar.malavali@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_dbg.c b/drivers/scsi/qla2xxx/qla_dbg.c
index d3c3d59..ee5c183 100644
--- a/drivers/scsi/qla2xxx/qla_dbg.c
+++ b/drivers/scsi/qla2xxx/qla_dbg.c
@@ -11,10 +11,11 @@
  * ----------------------------------------------------------------------
  * |             Level            |   Last Value Used  |     Holes |
  * ----------------------------------------------------------------------
- * | Module Init and Probe        |       0x0152       | 0x4b,0xba,0xfa |
+ * | Module Init and Probe        |       0x0159       | 0x4b,0xba,0xfa |
  * | Mailbox commands             |       0x1181       | 0x111a-0x111b  |
  * |                              |                    | 0x1155-0x1158  |
  * |                              |                    | 0x1018-0x1019  |
+ * |                              |                    | 0x1115-0x1116  |
  * |                              |                    | 0x10ca  |
  * | Device Discovery             |       0x2095       | 0x2020-0x2022, |
  * |                              |                    | 0x2011-0x2012, |
@@ -28,7 +29,7 @@
  * | DPC Thread                   |       0x4022       | 0x4002,0x4013  |
  * | Async Events                 |       0x5087       | 0x502b-0x502f  |
  * |                              |                    | 0x5047,0x5052  |
- * |                              |                    | 0x5040,0x5075  |
+ * |                              |                    | 0x5084,0x5075 |
  * |                              |                    | 0x503d,0x5044  |
  * | Timer Routines               |       0x6012       |                |
  * | User Space Interactions      |       0x70e1       | 0x7018,0x702e, |
@@ -41,6 +42,7 @@
  * |                              |                    | 0x70ad-0x70ae, |
  * |                              |                    | 0x70d1-0x70db, |
  * |                              |                    | 0x7047,0x703b |
+ * |                              |                    | 0x70de-0x70df, |
  * | Task Management              |       0x803d       | 0x8025-0x8026  |
  * |                              |                    | 0x800b,0x8039  |
  * | AER/EEH                      |       0x9011       |  |
@@ -54,10 +56,12 @@
  * |                              |                    | 0xb12c,0xb12d  |
  * |                              |                    | 0xb13a,0xb142  |
  * |                              |                    | 0xb13c-0xb140  |
+ * |                              |                    | 0xb149  |
  * | MultiQ                       |       0xc00c       |  |
  * | Misc                         |       0xd010       |  |
- * | Target Mode    |   0xe070       |  |
- * | Target Mode Management   |   0xf072       |  |
+ * | Target Mode    |   0xe070       | 0xe021  |
+ * | Target Mode Management   |   0xf072       | 0xf002-0xf003 |
+ * |                              |                    | 0xf046-0xf049  |
  * | Target Mode Task Management  |   0x1000b      |  |
  * ----------------------------------------------------------------------
  */
diff --git a/drivers/scsi/qla2xxx/qla_mr.c b/drivers/scsi/qla2xxx/qla_mr.c
index a3ca441..b97e701 100644
--- a/drivers/scsi/qla2xxx/qla_mr.c
+++ b/drivers/scsi/qla2xxx/qla_mr.c
@@ -1594,7 +1594,7 @@ qlafx00_timer_routine(scsi_qla_host_t *vha)
    */
   if (ha->mr.fw_critemp_timer_tick == 0) {
    tempc = QLAFX00_GET_TEMPERATURE(ha);
-   ql_log(ql_dbg_timer, vha, 0x6012,
+   ql_dbg(ql_dbg_timer, vha, 0x6012,
        "ISPFx00(%s): Critical temp timer, "
        "current SOC temperature: %d\n",
        __func__, tempc);
diff --git a/drivers/scsi/qla2xxx/qla_nx2.c b/drivers/scsi/qla2xxx/qla_nx2.c
index cbccb3f..8164cc9 100644
--- a/drivers/scsi/qla2xxx/qla_nx2.c
+++ b/drivers/scsi/qla2xxx/qla_nx2.c
@@ -3153,7 +3153,7 @@ qla8044_poll_flash_status_reg(struct scsi_qla_host *vha)
   ret_val = qla8044_rd_reg_indirect(vha, QLA8044_FLASH_STATUS,
       &flash_status);
   if (ret_val) {
-   ql_log(ql_log_warn, vha, 0xb120,
+   ql_log(ql_log_warn, vha, 0xb13c,
        "%s: Failed to read FLASH_STATUS reg.\n",
        __func__);
    break;
@@ -3484,7 +3484,7 @@ qla8044_write_optrom_data(struct scsi_qla_host *vha, uint8_t *buf,
   }
   erase_offset += QLA8044_SECTOR_SIZE;
  }
- ql_dbg(ql_dbg_user, vha, 0xb139,
+ ql_dbg(ql_dbg_user, vha, 0xb13f,
      "Got write for addr = 0x%x length=0x%x.\n",
      offset, length);
 
@@ -3653,7 +3653,7 @@ qla8044_clear_rst_ready(scsi_qla_host_t *vha)
   */
  drv_state &= ~(1 << vha->hw->portnum);
 
- ql_dbg(ql_dbg_p3p, vha, 0xb143,
+ ql_dbg(ql_dbg_p3p, vha, 0xb13d,
      "drv_state: 0x%08x\n", drv_state);
  qla8044_wr_direct(vha, QLA8044_CRB_DRV_STATE_INDEX, drv_state);
 }
@@ -3681,13 +3681,13 @@ qla8044_abort_isp(scsi_qla_host_t *vha)
   /* If IDC_CTRL DONTRESETHBA_BIT0 is set don't do reset
    * recovery */
   if (qla8044_idc_dontreset(ha) == DONTRESET_BIT0) {
-   ql_dbg(ql_dbg_p3p, vha, 0xb144,
+   ql_dbg(ql_dbg_p3p, vha, 0xb13e,
        "Reset recovery disabled\n");
    rval = QLA_FUNCTION_FAILED;
    goto exit_isp_reset;
   }
 
-  ql_dbg(ql_dbg_p3p, vha, 0xb145,
+  ql_dbg(ql_dbg_p3p, vha, 0xb140,
       "HW State: NEED RESET\n");
   qla8044_wr_direct(vha, QLA8044_CRB_DEV_STATE_INDEX,
       QLA8XXX_DEV_NEED_RESET);
diff --git a/drivers/scsi/qla2xxx/qla_sup.c b/drivers/scsi/qla2xxx/qla_sup.c
index 2f2e029..bd56cde 100644
--- a/drivers/scsi/qla2xxx/qla_sup.c
+++ b/drivers/scsi/qla2xxx/qla_sup.c
@@ -2944,7 +2944,7 @@ qla82xx_get_flash_version(scsi_qla_host_t *vha, void *mbuf)
   ha->fw_revision[0] = bcode[0x4];
   ha->fw_revision[1] = bcode[0x5];
   ha->fw_revision[2] = bcode[0x6];
-  ql_dbg(ql_dbg_init, vha, 0x015a,
+  ql_dbg(ql_dbg_init, vha, 0x0153,
       "Firmware revision %d.%d.%d\n",
       ha->fw_revision[0], ha->fw_revision[1],
       ha->fw_revision[2]);
-- 
1.7.1